static/locales/en.json

```
@@ -406,7 +406,67 @@
+    "cameraStreamFailed": "Failed to get camera stream",
+    "characterConfigFailed": "Failed to get character configuration",
+    "emotionTimeout": "Emotion analysis timeout",
+    "noAudioTrack": "No available audio track"
+  },
+  "camera": {
+    "iosPermissionHint": "Please check iOS Settings → Safari → Camera permission is set to \"Allow\"",
+    "notDetected": "No camera device detected",
+    "occupied": "Camera is occupied by another app? Close QR code/photo apps and try again"
+  },
+  "emotionManager": {
+    "title": "Live2D Emotion Mapping Manager",
```

Biome (2.3.14)

[error] The key emotionManager was already declared.

This is where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)